lib/jormungandr/test/integration/Main.hs

```diff
@@ -191,10 +189,10 @@ specWithServer tr = aroundAll withContext . after (tearDown . thd3)
         let setupContext wAddr nPort np = do
                 let baseUrl = "http://" <> T.pack (show wAddr) <> "/"
                 logInfo tr baseUrl
-                let sixtySeconds = 60*1000*1000 -- 60s in microseconds
+                let fiveMinutes = 300*1000*1000 -- 5min in microseconds
```

Comment:
Yolo ?

Reply:
:sweat_smile:  It's needed for this test -> https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Wallets.hs#L455-L475.